docs/usage.rst

```diff
@@ -161,6 +161,47 @@ functions all allow passing additional data to be tagged onto the error.
         Raven.setExtraContext({ foo: "bar" })
 
 
+.. _raven-js-recording-breadcrumbs:
+
+Recording Breadcrumbs
+---------------------
+
+Breadcrumbs are browser and application lifecycle events that are helpful in understanding the state of the application
+leading up to a crash.
+
+By default, Raven.js instruments browser built-ins and DOM events to automatically collect a few useful breadcrumbs
+for you:
+
+  * XMLHttpRequests
+  * URL / address bar changes
+  * UI clicks and keypress DOM events
+  * console log statements
+  * previous errors
+
+You can also record your own breadcrumbs:
+
+.. code-block:: javascript
+
+   Raven.captureBreadcrumb({
```

Comment:
Is there anything other than message and data someone might want to include here? Maybe link to https://docs.sentry.io/clientdev/interfaces/breadcrumbs/ for info about level, category, etc